mod_reforged/hooks/config/strings.nut

```diff
@@ -186,6 +186,21 @@ local vanillaDescriptions = [
  			}]
 	 	}),
 	},
+	{
+		ID = "perk.anticipation",
+		Key = "Anticipation",
+		Description = ::UPD.getDescription({
+			Fluff = "I saw these coming!",
+	 		Effects = [{
+ 				Type = ::UPD.EffectType.Passive,
+ 				Description = [
+					"Take reduced damage from the first " + ::MSU.Text.colorGreen(2) + " attacks you receive each battle.",
+					"This reduction is equal to your current [Ranged Defense|Concept.RangeDefense] + an additional " + ::MSU.Text.colorGreen("10%") + " for each tile between the attacker and you."
```

Comment:
Please remove the 10% for each additional tile part. I believe it is too strong; the perk is going to be very good even without it.

Reply:
From my testing it's the most essential part of this perk and not too strong. 
Please keep in mind that this only prevents the **first two attacks** in each battle and it quite literally follows in the footsteps of the old Anticipation, offering Player and Enemy alike a protection against getting sniped from far away.

Reply:
Oh, I thought each bro has like 35 base Ranged Defense. I was confusing it with Ranged Skill in my mind. Then disregard my comment - your design seems fine.